### DISCUSSION TITLE
Mouse Reference

### DISCUSSION THREAD

**Discussion by rpete** [cBioPortal/cbioportal#9108] [Q&A]
https://github.com/cBioPortal/cbioportal/blob/aa7aec069e892890ea9d221688969f177b1e83f9/core/src/main/scripts/importer/cbioportal_common.py#L891

Just curious why no mm10 is listed in the list? Would you not need mm10 to properly import a mouse study to map with mouse reference genome?

**Comment by jjgao** [comment]
@pvannierop Not sure if you are the right person to answer this, but I am wondering if you know whether we support mouse genome for CN segments?
